sphinx_gallery/backreferences.py

```diff
@@ -133,6 +133,8 @@ def scan_used_functions(example_file, gallery_conf):
     return backrefs
 
 
+# XXX This figure:: uses a forward slash even on Windows, but the op.join's
+# elsewhere will use backslashes...
```

Comment:
Just for discussion, out of the scope of this PR. But I think paths inside the rst file can all be forward slashes, even on Windows(I can't test it, don't own a windows machine,). Backreferences rely on paths relative to the project home, thus the leading forward slash. I wonder if in backreferences.py we shall join paths with forward slash instead of op.path.join